stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -284,17 +284,20 @@ def _apply_strategy_packaging(self, quants):
         def is_greater_eq(value, other):
             return float_compare(value, other, precision_rounding=rounding) >= 0
 
-        for location, location_quants in quants_per_bin:
-            location_quantity = sum(location_quants.mapped("quantity")) - sum(
-                location_quants.mapped("reserved_quantity")
-            )
-            if location_quantity <= 0:
+        # first search for the largest packaging in all locations
+        for pack_quantity in packaging_quantities:
```

Comment:
to remove any confusion, can you rename to packaging as it's not a pack
```suggestion
        for packaging_quantity in packaging_quantities:
```

Reply:
Renamed